src/officehours_api/serializers.py

```diff
@@ -266,4 +266,7 @@ def validate(self, attrs):
         hosts = queue.hosts.all()
         if attrs.get("assignee") and attrs["assignee"] not in hosts:
             raise serializers.ValidationError("Assignee must be a host!")
+        backend_type = attrs["backend_type"]
+        if backend_type not in queue.allowed_backends:
+            raise serializers.ValidationError("Invalid backend type!")
```

Comment:
Nice work. I'll review this more thoroughly next week. Before that, could you update this validation error message to be more detailed? Something like `f"{backend_type} is not one of the queue's allowed backend types (f{queue.allowed_backends})"`

Reply:
Of course! The error message has been updated in the commit ([Changed Error Output](https://github.com/tl-its-umich-edu/remote-office-hours-queue/pull/400/commits/9a04f254686ff5dd59356252b8d904599a31ebb3)).